kava-stats.js

```diff
@@ -766,7 +788,33 @@ const setDisplayValueById = (cssId, value) => {
 };
 
 const updateDisplayValues = async (denoms) => {
-    const [
+  const data = {
```

Comment:
this data object makes it easier to destructive responses, with Promise.All it had to be in exact order which made it hard to keep track & add new endpoints.